cmd/cbft/init_mem.go

```diff
@@ -77,6 +86,32 @@ func initMemOptions(options map[string]string) (err error) {
 	return nil
 }
 
+// Returns total memory based on cgroup limits, if possible.
+func getTotalMemStats() (uint64, error) {
+	stats, err := NewSystemStats()
```

Comment:
Why aren't you calling the Close() handle method on this stats obj? 
Isn't it required?